tests/vessels/test_generator.toit

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2022 Toitware ApS.
+// Use of this source code is governed by a Zero-Clause BSD license that can
+// be found in the tests/LICENSE file.
+
+import host.file
+
+DATA_256 := ((List 256: it).map: "$(%3d it), ").join ""
+
+main args:
+  out_file := args[0]
+  size := int.parse args[1]
+
+  data := "#["
```

Comment:
Add a comment that explains what this does and why. It isn't super easy to grok what it builds and does.

Reply:
sorry. should have done this without your request.
done.